test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -18,18 +76,37 @@ namespace Test::KozaniManagerRuntimeTests
         TEST_CLASS_SETUP(ClassSetup)
         {
             ::TB::Setup();
+            ::TP::AddPackage(KozaniManagerTestPackage::c_PackageDirName, KozaniManagerTestPackage::c_PackageFullName);
             return true;
         }
 
         TEST_CLASS_CLEANUP(ClassCleanup)
         {
+            ::TP::RemovePackageIfNecessary(KozaniManagerTestPackage::c_PackageFullName);
             ::TB::Cleanup();
             return true;
         }
 
-        // TODO Replace this with your own tests
-        TEST_METHOD(JustDoIt)
+        TEST_METHOD(ActivateRemoteApplicationTests)
         {
+            try
+            {
+                winrt::Microsoft::Kozani::ManagerRuntime::ManagerRuntimeManager runtimeManager;
+                auto statusCallback{ winrt::make_self<MyKozaniStatusCallback>() };
+
+                runtimeManager.ActivateRemoteApplication(winrt::Windows::ApplicationModel::Activation::ActivationKind::Launch,
+                    L"Microsoft.WindowsCalculator_8wekyb3d8bbwe!App", L"c:\\data\\connection.rdp", L"c:\\data\\additionalSettings.txt", 
+                    nullptr,    // IActivatedEventArgs
+                    statusCallback.as<winrt::Windows::Foundation::IInspectable>());
+
+                VERIFY_IS_TRUE(statusCallback->IsActivated(), L"IKozaniStausCallback::OnActivated() should have been called.");
```

Comment:
Is the message needed or redundant?

Reply:
The message will explain what is expected and when the condition is not met and the test fails, it will tell the dev what is the expected behavior.

Reply:
Just wondering how useful that is here. Happily deferring to your judgement

Reply:
Yes, it is very useful for people to understand what a test failure means.